metrics/prometheus_reporter.go

```diff
@@ -344,6 +345,11 @@ func (p *PrometheusReporter) ReportGauge(metric string, labels map[string]string
 	return constants.ErrMetricNotKnown
 }
 
+// ReportEvent is not implemented by PrometheusReporter
+func (p *PrometheusReporter) ReportEvent(title string, text string) error {
+	return xerrors.Errorf("Event reporting is not supported by PrometheusReporter")
```

Comment:
For not implemented methods, return the error that's defined in the constants package.
e.g.
https://github.com/topfreegames/pitaya/blob/2829a35ad96b87812b2b3041c4b40fb3576fe1ce/cluster/grpc_rpc_client.go#L153-L155

Reply:
Fixed in latest commit